ProjectShop/order/views.py

```diff
@@ -0,0 +1,28 @@
+from django.shortcuts import render
+from rest_framework import generics
+from rest_framework.permissions import IsAuthenticated, IsAdminUser
+
+from order.models import Order
+from order.serializers import OrderDetailSerializer
+from order.serializers import OrderListSerializer
```

Comment:
I guess,
from order.serializers import OrderDetailSerializer, OrderListSerializer
From new line we import like this:
import os
import re
import datetime